db/app.controller.js

```diff
@@ -0,0 +1,25 @@
+const { selectTopics, readEndpoints, selectArticlesById } = require("./app.model")
+
+exports.getAllTopics = (req,res,next) =>{
+    selectTopics().then(data =>{
+       
+    res.status(200).send({topics:data})
+       
+    }).catch(err => {
+       next(err)
+    })
+}
+exports.getAllEndpoints=(req,res, next)=>{
+readEndpoints().then((result)=>{
+    const enpoints ={...result}
+
+})
+}
+exports.getArticleByArticleID=(req,res,next)=>{
+const {article_id} = req.params
+selectArticlesById(article_id).then(article =>{
+    res.status(200).send(article)
```

Comment:
Be sure to send back your data in an object with a descriptive key, in this case { articles : articles }.
This will tell users or front-end developers what they are expecting to be returned